android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/GlobalAccountManager.kt

```diff
@@ -54,7 +58,18 @@ internal object GlobalAccountManager {
             // First error, start our count.
             authErrorCountWithinWindow = 1
         } else if (timeSinceLastAuthErrorMs <= AUTH_CHECK_CIRCUIT_BREAKER_RESET_MS) {
-            // Error within the reset time window, increment the count.
```

Comment:
I would suggesting keeping the original comment above the increment line, but adding a single comment above the sync check explaining the case when we want to quit the execution. One more thing, a code comment is better when it's a high level description. Mentioning implementation details like `If this is for an operation wrapped by `handleFxaExceptions`, then quit` might make a comment obsolete when somebody decided to change the name of the function or refactor the general flow of execution.